packages/story-editor/src/app/media/utils/useProcessMedia.js

```diff
@@ -193,16 +193,20 @@ function useProcessMedia({
       };
 
       const onUploadStart = () =>
-        updateExistingElements(resourceId, {
+        updateExistingElements(canvasResourceId, {
```

Comment:
cc @swissspidy for visibility (since you're also doing some refactoring work in these files)